sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -4673,6 +4673,13 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val PYTHON_FILTER_PUSHDOWN_ENABLED = buildConf("spark.sql.python.filterPushdown.enabled")
+    .doc("When true, enable filter pushdown to Python datasource, at the cost of running " +
+      "Python worker one additional time during planning.")
```

Comment:
do we still have additional planning now?

Reply:
Currently this is still true. I have a separate PR to combine filter pushdown & plan read.

Reply:
https://github.com/apache/spark/pull/50340